### DISCUSSION TITLE
[Question]Analysis Channel Segment CIR?

### DISCUSSION THREAD

**Discussion by bluelancer** [NVlabs/sionna#291] [Q&A]
Dear Sionna,

I am writing regarding each geometric channel, and its respective CIR, is it possible to extract each segment's component?

For example, I am interested in the free space radiation component, reflection component and connected by another free space radiation until arrive at the Rx

Is there such an API for extracting such information from the summarized CIR?

Best
Yifei

**Comment by jhoydis** [comment]
Hi Yifei,

Could you be a little bit more specific about what information you would need precisely? I am not sure what you mean by segment.
Every path coefficient $a_i$ is the result of a sequence of interactions like the one you describe. What do you mean by "summarized" CIR?
